lib/middleware/proxy.js

```diff
@@ -110,6 +110,7 @@ function generateFRN(req, res, next) {
 function sendRequest(req, res, next) {
     var options = {
```

Comment:
A new entry about the fix involved in this PR should be added to CHANGES_NEXT_RELEASE. In addition, it could include the entry for PR https://github.com/telefonicaid/fiware-orion-pep/pull/142 (that was merged missing that).